test/integration/rotation/04-create-workload-entry

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+log-debug "creating registration entry..."
+docker-compose exec spire-server \
+    /opt/spire/bin/spire-server entry create \
+    -parentID "spiffe://domain.test/spire/agent/x509pop/$(fingerprint conf/agent/agent.crt.pem)" \
```

Comment:
curious, why the preference of x509pop over join token, exercise more of the attestor flow?
Might be worse calling out in a comment somewhere for future tinkerers.

Reply:
No reason other than we already had it in place for other demos. Maybe in a future PR (I've got some queued up), I'll add an explicit x509pop integration test and move this test to using join tokens.

Reply:
wfm, I do actually like that its not using join token for now since it exercises more of the attestor flow (there are logic paths exclusively for join tokens).